rasterio/rio/windows.py

```diff
@@ -0,0 +1,171 @@
+"""$ rio windows"""
+
+
+import json
+import logging
+import os.path
+
+import click
+import cligj
+
+import rasterio
+from rasterio.rio import options
+from rasterio.rio.helpers import write_features
+from rasterio.warp import transform_bounds
+
+
+logger = logging.getLogger('rio')
+
+
+class _Collection(object):
+
+    """For use with `rasterio.rio.helpers.write_features()`."""
+
+    def __init__(self, src, bidx, precision, geographic):
+
+        """Produce a series of GeoJSON polygon features from a `RasterReader()`
```

Comment:
@geowurster How about `Export raster dataset windows to GeoJSON polygon features.` ?